ChustaSoft.Tools.Authorization.TestCustom.WebAPI/Startup.cs

```diff
@@ -52,13 +52,21 @@ public void ConfigureServices(IServiceCollection services)
             //    .WithUserCreatedAction<CustomUser, CustomUserAction>()
             //    .WithSqlServerProvider<AuthCustomContext, CustomUser, CustomRole>(BuildConnectionString());
 
+            //services.RegisterExternalAuthentication(x => 
+            //    { 
+            //        x.AddExternalAuthorization(ExternalAuthenticationProviders.Google, new ExternalAuthenticationSettings { ClientId = "", ClientSecret = "" }); 
+            //    }); 
+
+
             /*
             * Example with appSettings:
             */
             services.RegisterAuthorization<CustomUser, CustomRole>(_configuration, "d5ab5e3f5799445fb9f68d1fcdda3b9f")
                 .WithUserCreatedAction<CustomUser, CustomUserAction>()
                 .WithSqlServerProvider<AuthCustomContext, CustomUser, CustomRole>(BuildConnectionString());
 
+            services.RegisterExternalAuthentication(_configuration);
```

Comment:
Same explained in the Basic example

Reply:
We are adding the AuthorizationController from the Chustasoft.ToolsAuthorization.AspNet assembly in both test projects with the .AddAuthorizationControllers() method